Sources/SwiftlyCore/HTTPClient.swift

```diff
@@ -88,21 +207,91 @@ public struct SwiftlyHTTPClient {
             return release
         }
 
-        return try await self.mapGitHubTags(limit: limit, filterMap: filterMap) { page in
-            try await self.getReleases(page: page)
+        guard !swiftOrgFiltered.isEmpty else {
```

Comment:
Is this really necessary?

Reply:
Agreed, this is not needed. I'll remove it.